deploy/provision.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+# Copyright (c) 2015 Intracom S.A. Telecom Solutions. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v1.0 which accompanies this distribution,
+# and is available at http://www.eclipse.org/legal/epl-v10.html
+# ------------------------------------------------------------------------------
+# Display distribution release version
+#-------------------------------------------------------------------------------
+
+BASE_DIR=$(pwd)
+cd $BASE_DIR
+
+# Prevent headers and kernels updates. Mininet will not work with Linux kernel
+# newer than 3.12.x
+#-------------------------------------------------------------------------------
+apt-mark hold linux-image-generic linux-headers-generic
+
+# NSTAT node necessary tools
+#-------------------------------------------------------------------------------
+apt-get update && apt-get install --force-yes -y \
+    git \
+    unzip \
+    wget \
+    iperf \
+    mz \
+    wireshark \
+    net-tools
+
+# CONTROLLER_node necessary tools
+#-------------------------------------------------------------------------------
+apt-get install --force-yes -y \
+    openssh-client \
+    openssh-server \
+    openjdk-7-jdk \
+    openjdk-7-jre
+
+# SOUTHBOUND_GEN_node necessary tools (build tools)
```

Comment:
create separate sections for "MT-Cbench node" and "Multinet node", and place the dependencies to the corresponding sections